config/routes.rb

```diff
@@ -7,4 +9,13 @@
   root 'certificates#index'
   get '/events', to: 'events#index'
   get '/events/:page', to: 'events#page'
+
+  get '/devauth' => 'dev_auth#show'
```

Comment:
Is this route necessary?

If it are, but it's just used in development mode then we should add a `Rails.env == :development` conditional

Reply:
It's for creating local users for testing, so for now we can leave it there, but I agree it should be constrained by the environment.